tmt/steps/provision/podman.py

```diff
@@ -274,7 +274,6 @@ def _run_ansible(
             *self._ansible_extra_args(extra_args),
             '-c', 'podman', '-i', f'{self.container},', playbook
             ])
-
```

Comment:
Several lines removed, unrelated change :)

Reply:
Fixed in 78a9ca02e80c1a06b93406367a342e8d83c0a30e.